test/library/packages/Sort/correctness/test-radix-bucketizer.chpl

```diff
@@ -58,8 +58,8 @@ proc testBucketizer() {
   }
 
   {
-    var i = 1;
-    for (idx,bin) in b.classify(A, 1, A.domain.high, criterion, 0) {
+    var i = 0;
+    for (idx,bin) in b.classify(A, 0, A.domain.high, criterion, 0) {
```

Comment:
Ideally, `A.domain.low` but it wasn't like that before either

Reply:
same applies to 5(?) other cases below

Reply:
https://github.com/chapel-lang/chapel/pull/14521/commits/00a6e2121a0e61f700c0c6e34b27dbedffc6eab1 does this if you could give it a quick sanity check.

Reply:
nice